aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -647,87 +942,125 @@ def export_tree(
-            progress_bar.update()
 
-            # Important to set create=False, otherwise creates twice a subfolder. Maybe this is a bug of insert_path?
-            thisnodefolder = nodesubfolder.get_subfolder(sharded_uuid, create=False, reset_limit=True)
+def export_zip(
+    entities: Optional[Iterable[Any]] = None,
+    filename: Optional[str] = None,
+    use_compression: bool = True,
+    **kwargs: Any,
+) -> Tuple[float, ...]:
+    """Export in a zipped folder
 
-            # Make sure the node's repository folder was not deleted
-            src = RepositoryFolder(section=Repository._section_name, uuid=uuid)  # pylint: disable=protected-access
-            if not src.exists():
-                raise exceptions.ArchiveExportError(
-                    f'Unable to find the repository folder for Node with UUID={uuid} in the local repository'
-                )
+    .. deprecated:: 1.2.1
+        Support for the parameters `what` and `outfile` will be removed in `v2.0.0`.
+        Please use `entities` and `filename` instead, respectively.
 
-            # In this way, I copy the content of the folder, and not the folder itself
-            thisnodefolder.insert_path(src=src.abspath, dest_name='.')
+    :param entities: a list of entity instances; they can belong to different models/entities.
 
-    close_progress_bar(leave=False)
+    :param filename: the filename
+        (possibly including the absolute path) of the file on which to export.
 
-    # Reset logging level
-    if silent:
-        logging.disable(level=logging.NOTSET)
+    :param use_compression: Whether or not to compress the zip file.
+
+    """
+    warnings.warn('function is deprecated, use `export` instead', AiidaDeprecationWarning)  # pylint: disable=no-member
+    output_data = export(
+        entities=entities,
+        filename=filename,
+        file_format=ExportFileFormat.ZIP,
+        use_compression=use_compression,
+        **kwargs,
+    )
+    if 'time_write_stop' in output_data:
+        return (output_data['time_extract_start'], output_data['time_write_stop'])
+
+    # there was no data to write
+    return (output_data['time_extract_start'], output_data['time_extract_stop'])
+
+
+def export_tar(
+    entities: Optional[Iterable[Any]] = None,
+    filename: Optional[str] = None,
+    **kwargs: Any,
+) -> Tuple[float, ...]:
+    """Export the entries passed in the 'entities' list to a gzipped tar file.
+
+    .. deprecated:: 1.2.1
+        Support for the parameters `what` and `outfile` will be removed in `v2.0.0`.
+        Please use `entities` and `filename` instead, respectively.
+
+    :param entities: a list of entity instances; they can belong to different models/entities.
+
+    :param filename: the filename (possibly including the absolute path)
+        of the file on which to export.
+    """
+    warnings.warn('function is deprecated, use `export` instead', AiidaDeprecationWarning)  # pylint: disable=no-member
```

Comment:
```suggestion
    warnings.warn('function is deprecated and will be removed in AiiDA v2.0.0, use `export` instead', AiidaDeprecationWarning)  # pylint: disable=no-member
```

Reply:
fixed in b689f5e6986c15ab18ad225e5659f1ef3bf8f9c0